packages/cspell/src/app/util/fileHelper.ts

```diff
@@ -36,6 +37,16 @@ export async function readConfig(
     configFile: string | CSpellConfigFile | undefined,
     root: string | undefined,
 ): Promise<ConfigInfo> {
+    const configPath = getEnvironmentVariable('CSPELL_CONFIG_PATH');
+    if (!configFile && configPath && existsSync(configPath)) {
+        configFile = configPath;
+    }
+
+    const defaultPath = getEnvironmentVariable('CSPELL_DEFAULT_CONFIG_PATH');
+    if (!configFile && defaultPath && existsSync(defaultPath)) {
+        configFile = defaultPath;
+    }
```

Comment:
I think it can be simpler:
```suggestion
    configFile ??= getEnvironmentVariable(environmentKeys.CSPELL_CONFIG_PATH);
    configFile ??= getEnvironmentVariable(environmentKeys.CSPELL_DEFAULT_CONFIG_PATH);
```

If someone specifies an environment variable that points to a nonexistent file, it should report the error. This also avoids the `existsSync` call.